packages/expo-notifications/src/PushTokenManager.web.ts

```diff
@@ -1,2 +1,15 @@
-// mock
-export default {};
+import { PushTokenManagerModule } from './PushTokenManager.types';
+
+let warningHasBeenShown = false;
+
+export default {
+  addListener: () => {
+    if (!warningHasBeenShown) {
+      console.warn(
+        '[expo-notifications] Notifications handling is not yet fully supported on web. Handling notifications will have no effect.'
```

Comment:
maybe just `  addListener: () => {}` for this one? this message seems specific to `NotificationsHandlerModule`.

Reply:
Changed the warning message to

> [expo-notifications] Listening to push token changes is not yet fully supported on web. Adding a listener will have no effect.

👍 Thank you for noticing this! 👁️